src/components/Card/src/index.ts

```diff
@@ -1,8 +1,8 @@
-import Card from './Card';
+import Card from './Card/Card';
 
 export default Card;
-export * from './Card';
-export * from './CardActions';
-export * from './CardContent';
-export * from './CardHeader';
-export * from './Paper';
+export * from './Card/Card';
+export * from './CardHeader/CardHeader';
+export * from './CardActions/CardAction';
+export * from './CardContent/CardContent'
```

Comment:
Do we actually want to export these? I agree this is the case for `CardProps` (`export *  from './Card/Card'`) and maybe `Paper` as that might be seen as a own component (even though it is quite useless atm). I don't really see a use-case why we want to expose all the components which are only meant to be used as part of the Card though.

Reply:
Good point, I have made some changes to only export the card and paper components for now.